src/scores/categorical/contingency_impl.py

```diff
@@ -848,8 +848,24 @@ def odds_ratio_skill_score(self) -> xr.DataArray:
         Returns:
             xr.DataArray: An xarray object containing the odds ratio skill score
 
+        .. math::
+
+            \\begin{aligned}
+                \\text{ORSS} &= \\frac{\\theta - 1}{\\theta + 1}
```

Comment:
I would suggest instead of \\theta I would use something list \\text{OR} tp make it just a little easier on new readers, or even  \\text{Odds Ratio}. What's there isn't wrong, but I'm trying  to make it as easy to read as possible. Happy to hear if you think differently, but that's my thought on it.

Reply:
`theta` is what was used in the reference, but I have no preference either way. I can change it to `OR` if you prefer.

Reply:
Please change it to OR.

Reply:
don